### ISSUE TITLE
Don't work on windows: WebDriverError: unknown error: cannot find Chrome binary

### ISSUE THREAD

**Issue by jrichardsz** [usil/seleniumjs4all#14]

**Comment by jrichardsz** [comment]
https://stackoverflow.com/questions/73628163/handling-selenium-webdrivers-on-node-js-when-sigint-is-recieved

**Comment by jrichardsz** [comment]
```
{
    "arguments": [
      "--log-level=1",
      "--no-sandbox",
      "--disable-gpu",
      "--disable-dev-shm-usage",
      "--headless",
      "--window-size=2200,1600"
    ],
    
    "options": {
        "excludeSwitches" :  "...",
        "setChromeBinaryPath" : "..."
    }
  }
```

https://github.com/usil/seleniumjs4all/blob/fix-CustomEnviroment/src/browsers/browserDriver.js#L26

```
chromeOptions[methodName](methodParam);
```

**Comment by jrichardsz** [comment]
add a link with all the available options

**Comment by jrichardsz** [comment]
https://stackoverflow.com/questions/912596/how-to-turn-a-string-into-a-javascript-function-call

**Comment by iSkyNavy** [comment]
This bug was solved in the commit #15 .
The list of chrome options is 
![image](https://user-images.githubusercontent.com/66818290/224855422-33f64df0-9ee0-4105-8827-55a9759613a4.png)
![image](https://user-images.githubusercontent.com/66818290/224855600-d663376f-883b-41d3-8b90-a04e217359fd.png)
